express/src/lib/validators/emailValidator.ts

```diff
@@ -0,0 +1,23 @@
+import {validationResult} from "./validationResult";
+import isRfc822Compliant from "../rfc822-validate";
+import allowedEmailDomains from "../allowedEmailDomains";
+
+export async function validateEmail(emailAddress: string): Promise<validationResult> {
```

Comment:
```suggestion
export async function validateEmail(emailAddress: string): Promise<validationResult> {
```

In a later iteration we'll need to pre-load the domains at startup so we can avoid the Promise here.